app/api/register/route.ts

```diff
@@ -0,0 +1,37 @@
+import { NextResponse } from "next/server";
+// for sending an email verification message
+// import crypto from 'crypto';
+// import { sendVerificationEmail } from '@/app/lib/sendEmail';
+
+export async function POST(request: Request) {
+    try {
+        // Recieve the email and password of the user
+        const { email, password } = await request.json();
+
+        // Check valid email format using regex
+        const emailRegex = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g;
```

Comment:
very close! but try this regex:
```suggestion
        const emailRegex = /^[a-zA-Z0-9._%+-]+@([\w-]+\.)+[\w-]{2,4}$/g
```
this clears numeric characters as well.